test/e2e/suite/command/manifest_index.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	"oras.land/oras/test/e2e/internal/testdata/multi_arch"
+	. "oras.land/oras/test/e2e/internal/utils"
+)
+
+var _ = Describe("ORAS beginners:", func() {
+	When("running manifest index command", func() {
+		When("running `manifest index create`", func() {
+			It("should show help doc with alias", func() {
+				ORAS("manifest", "index", "create", "--help").MatchKeyWords("Aliases", "pack").Exec()
+			})
+		})
+	})
+})
+
+var _ = Describe("1.1 registry users:", func() {
+	When("running `manifest index create`", func() {
+
+		It("should create index by using source manifest digests", func() {
+			ORAS("manifest", "index", "create", RegistryRef(ZOTHost, ImageRepo, "index-create-by-digest"),
+				string(multi_arch.LinuxAMD64.Digest), string(multi_arch.LinuxARM64.Digest)).
+				MatchKeyWords("Fetched", "Pushed",
+					"sha256:cce9590b1193d8bcb70467e2381dc81e77869be4801c09abe9bc274b6a1d2001").Exec()
```

Comment:
Is this digest of the crafted index? Better add metadata prmopt like `Digest: xxxx` so the spec is more readable.